src/gt4py/loader.py

```diff
@@ -52,7 +52,7 @@ def load_stencil(
         raise ValueError("Invalid frontend specification ({name})".format(name=frontend_name))
 
     builder = StencilBuilder(
-        definition_func, options=build_options, backend=backend_cls, frontend=frontend
```

Comment:
This is not really needed after the latest change, but I don't want to trigger another 90 min CI run to role back, before the strategy in general is approved.